src/EnergyPlus/ElectricPowerServiceManager.cc

```diff
@@ -2294,6 +2364,12 @@ namespace EnergyPlus {
 		return aCEnergyOut_;
 	}
 
+	DCtoACInverter::InverterModelType
+	DCtoACInverter::modelType() const
+	{
+		return modelType_;
+	}
```

Comment:
I do feel concerned about unnecessary use of getters and setters.  I love using getters and setters in library code where you are concealing the underlying simulation logic.  In EnergyPlus, where our goal should always be to maximize performance, I don't want arbitrary getters/setters to accidentally, and unnecessarily, take clock cycles.

Reply:
Maybe I've spend too much time in OpenStudio development where prolific getters and setters are the reason for getting out of bed in the morning. They're not generally being used, except in the unit tests. It makes it so I don't have to make all the member variables public for inspection. If it's that important to you I can remove these and make the member variables public or protected with the test fixture being a friend class.